QUEST.tsv

```diff
@@ -179,7 +179,7 @@ QUEST_20150317_000178	NPC HIDE/UNHIDE Setting!
 QUEST_20150317_000179	Follower Algis
 QUEST_20150317_000180	This is the moment the Goddess foretold to the first Paladin. {nl}Come. Ask me anything.
 QUEST_20150317_000181	A long, long time ago, there were nomads that suffered from a cursed plague. {nl}The honorable first Paladin removed the curse with a Holy Relic.
-QUEST_20150317_000182	The Watchers are the descendants of those nomads. {nl}However, the Holy Relic was not meant to be used for healing those people.
+QUEST_20150317_000182	The watchers are the descendants of those nomads. {nl}However, the Holy Relic was not meant to be used for healing those people.
```

Comment:
Hmm, it wasn't consistent.  I'll change the other 2 to capitalized as well then.